### ISSUE TITLE
Update ops from BatchNorm layer might not be handled

### ISSUE THREAD

**Issue by yoshihikoueno** [jobpasin/tooth-1d#6]
https://github.com/jobpasin/tooth-1d/blob/1377e7a49003808037feb025f5ab208ee364f464/coor_model.py#L260

From what I understand,
1. if you use batchnorm from keras API, you need to explicitly add ops to update BatchNorm ops to UPDATE_OPS set since keras API won't do it for you.

2. UPDATE_OPS should be appended to train_op so that they actually get executed.

References
https://github.com/tensorflow/tensorflow/issues/25525
https://stackoverflow.com/questions/55421984/batch-normalization-in-tf-keras-does-not-calculate-average-mean-and-average-vari

**Comment by jobpasin** [comment]
Thanks for the info. Currently using tf.layers.batch_normalization for the time being.
